src/game.cpp

```diff
@@ -3285,8 +3285,11 @@ void game::debug()
             }
 
             auto rt = m.route( u.pos(), *dest, u.get_pathfinding_settings(), u.get_path_avoid() );
-            u.set_destination( rt );
-            if( !u.has_destination() ) {
+            
+            if(rt.size() > 0) {
```

Comment:
```suggestion
            if( rt.size() > 0 ) {
```

Reply:
Please use `empty()` (as in `if( !rt.empty() )`) as you're not interested in the size, only in the emptiness.